config/ocp/kustomization.yaml

```diff
@@ -9,7 +9,9 @@ namespace: flotta
 # Note that it should also match with the prefix (text before '-') of the namespace
 # field above.
 namePrefix: flotta-operator-
-
+images:
+- name: "gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0"
```

Comment:
I see there is [v0.12.0](https://github.com/brancz/kube-rbac-proxy/releases/tag/v0.12.0) recently but last image version is v0.11.0. Do you think it makes sense to update?

Reply:
The 0.8.0 version is quite old (november 2020) so maybe there was a particular reason to use this version, or maybe it was just the reuse of another project/example. @jakub-dzon can you provide feedback on that?

Reply:
When I look at the version referred by kubebuilder, there is a mixture of v0.8.0 and v0.11.0
https://github.com/kubernetes-sigs/kubebuilder/search?q=kube-rbac-proxy

The documentation seems a bit outdated as well.
By looking at https://github.com/brancz/kube-rbac-proxy release notes, there shouldn't be any compatibility issues, while there is a support in tls 1.3. 
Therefore I recommend to move to v0.11.0